templates/rhpam70-trial-ephemeral.yaml

```diff
@@ -159,17 +97,11 @@ parameters:
 - displayName: Maven repository URL
   description: Fully qualified URL to a Maven repository.
   name: MAVEN_REPO_URL
```

Comment:
@bmozaffa Does it have sense to allow definition of Maven repo URL in this template? AFAIK the Kie server should use just repo from Workbench which is defined by MAVEN_REPO_SERVICE in Deployment config.

Reply:
@sutaakar No, I think you're right and it doesn't make sense, thanks for spotting the error. However, I'm trying to figure out how to deal with a dependency that's in an external Maven. Might be better to wait and figure that out first, as it would affect how this should be addressed.

Reply:
@sutaakar This is now addressed by https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/38, though it might yet be refactored further after other PRs merge.